cmd/juju/model/setcredential.go

```diff
@@ -171,12 +171,28 @@ func (c *modelCredentialCommand) findCredentialLocally(ctx *cmd.Context) (*cloud
 	return credential, nil
 }
 
-const modelCredentialDoc = `This command changes the cloud credential used by a model.
+const modelCredentialDoc = `
+This command links a credential cached on a controller to a specific model. It
+does not change/update the contents of an existing active credential. See
+command ` + "`update-credential`" + ` for that.
 
-When current user has specified credential for the given cloud remotely, i.e
-it exists on the controller, Juju will use it to replace model's credential.
+The credential specified may exist locally (on the client), remotely (on the
+controller), or both. The command will error out if the credential is neither
+remotely nor locally stored.
 
-If the specified credential does not exist remotely on the controller, but does exist locally on the client,
-Juju will upload it to the controller first and then replace the model's credential.
+When remote, the credential will be linked to the specified model.
 
+When local and not remote, the credential will first be uploaded to the
+controller and then linked.
```

Comment:
We do have IT-savy users. I am not sure that "link" is the technical jargon I'd want to use in this context - reference is better for our audience?..